### DISCUSSION TITLE
Dropdown Menu Vs Select Component

### DISCUSSION THREAD

**Discussion by thisisankur0** [radix-ui/primitives#2627] [Help] [answered]
Hey, wanted to know if there is any inherent difference between using Dropdown menu or Select component, working on a project where a dropdown menu is being used inside a form for selecting values. Any potential drawbacks?

**Comment by benoitgrelard** [comment, marked as answer]
Yes, definitely the wrong pattern. They have completely different semantics.
- `DropdownMenu` is a UI pattern to present a list of actionnable items
- `Select` is a UI pattern to allow users to pick a value out of a set of options
